samples/drivers/lora/receive/prj.conf

```diff
@@ -2,5 +2,4 @@ CONFIG_LOG=y
 CONFIG_SPI=y
 CONFIG_GPIO=y
 CONFIG_LORA=y
-CONFIG_LORA_SX12XX=y
```

Comment:
This should probably be squashed to previous commit in order to preserve bisectability.

Reply:
Done